mypy.ini

```diff
@@ -27,3 +26,19 @@ django_settings_module = scripts.drf_tests_settings
 
 [mypy-uritemplate.*]
 warn_unreachable = false
+
+[mypy-yaml.*]
+disallow_untyped_defs = false
+disallow_incomplete_defs = false
```

Comment:
I think these are necessary due to the way `pytest-mypy-plugins` works.

Mypy ordinarily ignores errors from site-packages imports, but they aren't ignored in tests.

Reply:
This can be a new bug / feature request in `pytest-mypy-plugins`. Users should not see this.

Reply:
👍 Reported issue https://github.com/typeddjango/pytest-mypy-plugins/issues/134